src/UI/InputHandler.java

```diff
@@ -3,11 +3,16 @@
 import java.awt.event.KeyEvent;
 import java.util.ArrayList;
 import java.util.Arrays;
-public class InputHandler
+import java.util.HashMap;
+
+import Engine.IController;
+import Engine.OptionSelector;
+import UI.InputHandler.InputAction;
+public class InputHandler implements IController
```

Comment:
Hmm. I can't say I expected to see this, but It does make a certain amount of sense as a way to keep input-specifics in this one class to abstract it away from the rest of the codebase.

Reply:
Well, I didn't expect to do this, but then it kinda happened.

Reply:
Hmm. I feel like the static and non-static portions aren't particularly closely-bound.
It is amusing/novel to leave them together, though, so I guess I don't care much either way.

Reply:
I started out by making a separate class, but it did nothing but make calls to InputHandler anyway.
I imagine I could find a more structured way to compose it if I tried, but it's simple enough and works well enough now that it doesn't seem worth the effort to try and overthink it at this point.